cmd/http/config.json

```diff
@@ -1,26 +1,34 @@
 {
     "ConsistentHash": "rendezvous",
     "BucketSize": 10,
-    "ReplicaNum": 3,
+    "ReplicaNum": {
+        "Local": 2,
```

Comment:
do we still need local and remote config?

Reply:
The definition is at line 10  "//" : "The threshold is to check if a store is located in a remote datacenter. It is the latency in millis seconds to set up connection",

As we discussed, we need to config the number of LOCAL nodes for synchronized  write and the REMOTE nodes for asynchronized write.

Reply:
discussed offline. no issue here.